card/spec/lib/card/query_spec.rb

```diff
@@ -397,8 +396,8 @@
 
   describe "relative" do
     it "should clean wql" do
-      wql = Card::Query.new( :part=>"_self",:context=>'A' )
-      expect(wql.query[:part]).to eq('A')
+      query = Card::Query.new( :part=>"_self",:context=>'A' )
+      expect(query.statement[:part]).to eq('A')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.